docs/faq/data-loading.md

```diff
@@ -44,6 +51,15 @@ We generally recommend setting the incremental loads to every hour to help limit
 
 Unfortunately not yet.
 
+## **Do you support change data capture \(CDC\) or logical replication for databases?**
+
+We currently support [CDC for Postgres 10+](../integrations/sources/postgres.md). We are adding support for a few other databases April/May 2021.
```

Comment:
We also support MySql. I would link the CDC page here instead of the postgres page.